core/shared/src/main/scala/zio/prelude/Associative.scala

```diff
@@ -1315,30 +1315,30 @@ trait AssociativeSyntax extends PlatformSpecificAssociativeSyntax {
    * Provides infix syntax for combining two values with an associative
    * operation.
    */
-  implicit class AssociativeOps[A](l: A) {
+  implicit class AssociativeOps[+A](l: A) {
```

Comment:
I thought this isn't necessary, no other Ops classes have this :thinking: Is there an example  where this is needed?

Should we adjust the other Ops classes to also have covariant type parameters?